### ISSUE TITLE
Cannot read property 'value' of undefined at new deviceMakerGarage

### ISSUE THREAD

**Issue by degofedal** [bwp91/homebridge-platform-wemo#252]
### What issue do you have? Please be as thorough and explicit as possible.
Wemo Maker (Garage Door) is unresponsive in the Home app. Homebridge log throws this error with Wemo plugin versions 3.0.1, 3.0.2b3 and 3.0.2. Version 2.15.2 is fine and works as expected

**Homebridge log:**
```
[Wemo] [Garage Door] could not be initialised as Cannot read property 'value' of undefined at new deviceMakerGarage (/homebridge/node_modules/homebridge-platform-wemo/lib/device/maker-garage.js:66:80).
```

### Details of your setup.
Homebridge 1.3.4, Config UI latest
Node.js v14.16.1
NPM v6.14.12
Dockerized Homebridge on Synology NAS

* Which version of this plugin (homebridge-platform-wemo) do you have? Has the issue started since upgrading from a previous version?

Error shows up with Wemo plugin versions 3.0.1, 3.0.2b3 and 3.0.2

* Which Wemo devices do you have that are causing issues? Please include product models if applicable.
Wemo Maker

### Please paste any relevant logs below.
```
[Wemo] [Garage Door] could not be initialised as Cannot read property 'value' of undefined at new deviceMakerGarage (/homebridge/node_modules/homebridge-platform-wemo/lib/device/maker-garage.js:66:80).
```

**Comment by bwp91** [comment]
Hi @degofedal 
Could you update to beta version  3.0.3-beta.0 to see if this helps?
Thanks

**Comment by degofedal** [comment]
Hi @bwp91 

Wow thanks for the quick response. Just did the update. The log i clean (no errors) but the Garage Door (Wemo Maker) is unresponsive in the Home app. Let me know if there is anything else you want me to try. Happy to help.

**Comment by bwp91** [comment]
Hi @degofedal

>  the Garage Door (Wemo Maker) is unresponsive in the Home app

is this still happening? With the new update, accessories in the home app should appear as 'No Response' until the corresponding device is discovered by the plugin, then the 'No Response' should go away again.

If the accessory continues to display 'No Response' even after the device is discovered then of course this is something I'll need to look into further!

Thanks

**Comment by degofedal** [comment]
> Hi @degofedal
> 
> > the Garage Door (Wemo Maker) is unresponsive in the Home app
> 
> is this still happening? With the new update, accessories in the home app should appear as 'No Response' until the corresponding device is discovered by the plugin, then the 'No Response' should go away again.
> 
> If the accessory continues to display 'No Response' even after the device is discovered then of course this is something I'll need to look into further!
> 
> Thanks

Hi @bwp91 

Yes, the garage door is stil unresponsive in Home app with version 3.0.3-beta.1. The log however show that it should be fine.

```
[4/14/2021, 10:34:40 AM] [Wemo] Launched child bridge with PID 4890

...

[4/14/2021, 10:34:40 AM] [Wemo] Loaded homebridge-platform-wemo v3.0.3-beta.1 child bridge successfully
[4/14/2021, 10:34:40 AM] Loaded 1 cached accessories from cachedAccessories.0E21025AA60F.
Preparing Advertiser for 'Wemo 1103' using ciao backend!
[4/14/2021, 10:34:40 AM] [Wemo] [v3.0.3-beta.1] initialised. Setting up device discovery.
[4/14/2021, 10:34:41 AM] [Wemo] ✓ Setup complete. Thanks for using this plugin, I hope you find it helpful!
Starting to advertise 'Wemo 1103' using ciao backend!

...


[4/14/2021, 10:35:11 AM] [Wemo] [Garage Door] initialised with s/n XXXXXXXXXXXX and ip/port 192.168.0.13:49153
[4/14/2021, 10:35:11 AM] [Wemo] [Garage Door] current state [closed].
```

**Comment by degofedal** [comment]
> Hi @degofedal
> 
> > the Garage Door (Wemo Maker) is unresponsive in the Home app
> 
> is this still happening? With the new update, accessories in the home app should appear as 'No Response' until the corresponding device is discovered by the plugin, then the 'No Response' should go away again.
> 
> If the accessory continues to display 'No Response' even after the device is discovered then of course this is something I'll need to look into further!
> 
> Thanks

Argh so sorry for wasting your time. I just noticed as i posted my previous response that i had one cached accessory. I removed it via Homebridge settings and now all is well 👍 Problem solved. Thanks again for your quick response

**Comment by bwp91** [comment]
Fab. Just released 3.0.3 with the fix for the original Maker issue.
Let me know if any more problems in the future!

**Comment by julien94800** [comment]
Hello,
Same problem from my side. Just upgrade from V2 to V3 and now in home app the maker is unresponsive, whereas in homebridge log everything seems to be ok. Do we have to remove old one and reintegrate for migration?
Thanks in advance

**Comment by bwp91** [comment]
Hi @julien94800 
I’ve just published another beta version that should solve this, could you give it a try?
It’s v3.0.4-beta.0

**Comment by julien94800** [comment]
Hi @bwp91 
I confirm it's now online again after upgrading to beta version!
Thanks very much for your reactivity and support!
Your job on this plugin is really awesome!!!!!!

**Comment by bwp91** [comment]
Just released v3.0.4 with all the fixes for the Maker :)

**Comment by julien94800** [comment]
It works perfectly !!
Thanks again very much
